src/smc-webapp/r_misc.cjsx

```diff
@@ -1329,6 +1329,42 @@ exports.UpgradeAdjustor = rclass
     render_addon: (misc, name, display_unit, limit) ->
         <div style={minWidth:'81px'}>{"#{misc.plural(2,display_unit)}"} {@render_max_button(name, limit)}</div>
 
+    remove_upgrade_from_all: (projects_with_this_upgrade, name) ->
+        for project_id in projects_with_this_upgrade
+            @remove_upgrade_from_project(project_id, name)
+
+    render_remove_upgrade_from_all: (projects_with_this_upgrade, name) ->
+        <MenuItem
+            eventKey="0"
+            onClick={=>@remove_upgrade_from_all(projects_with_this_upgrade, name)}>
+            All
+        </MenuItem>
+
+    remove_upgrade_from_project: (project_id, name) ->
+        quotas_to_apply = redux.getStore('projects').get_projects_upgraded_by()[project_id]
+        delete quotas_to_apply[name]
+        redux.getActions('projects').apply_upgrades_to_project(project_id, quotas_to_apply)
```

Comment:
This looks like this removes the upgrade when it gets selected. I think expected behavior is for all the removals to happen when the user saves the changes.